app/controllers/application_controller.rb

```diff
@@ -85,6 +85,7 @@ def authenticate_token
     authenticate_with_http_token do |token, _options|
       @session = Session.find(token)
       return false if @session.nil?
+      Rails.logger.info("Logged in user with id #{@session.uuid}")
```

Comment:
@omgitsbillryan OK, I changed it so that it will only log the UUID and token once when the user initially logs in rather than with each request.